### PR TITLE
Mexican

### PR DESCRIPTION
**Pull request by lukekivi**
TORTILLAS MY NINJA

### CODE REVIEW

File: Lucas/Italian/sugo_di_pomodoro-marinara.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  
+## Instructions:
+1. In a pot brown the garlic in oil. Just until it is golden. Do not let parts of the garlic become **dark** brown or black.
+2. Add tomatoes. 
+3. Add hot stuff. I prefer fresnos, which I just smash once or slice in half. You just need to allow liquid to penetrate the skin since you will not cook for a long time.
+4. Add your sprig of basil to the top and allow it to wilt in to the sauce.
+5. Stew at a medium to medium-low heat until the oil is orange in color.
+6. Remove larges stems. Blend until you reach desired chunkiness. I like smooth with a few chunks.
+7. (optional) Return to pot. As far as seasoning goes you are just building a base. No need to season to final levels, this is done in pan before serving. Balsamic vinegar reduction is **NOT** traditional but I like to add a little. Maybe 3 dashes. It brings depth and a sweetness that begins to round out the flavor profile of the sauce.
+8. (optional) CAN SOME! Flavors develop over time so even if you do not can, flavor peaks a few days after cooking.
```
Comment (AndrewHaisting) [inline Lucas/Italian/sugo_di_pomodoro-marinara.md:21]: Whoops! This stuff shouldn't be in this changeset. 

There's lots of ways to solve this, but the root problem is that the commits on `main` don't match the commits you have here. Lots of ways to solve that, though we should probably hop on a call to sort it out, as it its a bit tricky.

File: Lucas/Mexican/flourTortillas.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Flour Tortillas
+Makes 12 8in diameter tortillas.
+## Ingredients
+- **3/4 lb** (2.75 C) All-purpose flour
+- **5 T** Animal fat (lard, duck fat, beef fat)
```
Comment (AndrewHaisting) [inline Lucas/Mexican/flourTortillas.md:5]: Consider adding a note here about substituting vegetable fat, or blending multiple types of fat.

File: Lucas/Mexican/flourTortillas.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- **3/4 t** Salt
+- **3/4 C** Very warm water
+
+## Instructions
+1. Combine flour and fat. Use your fingers in a large bowl, spin in a food processor or use a paddle attachment for a mixer until completely incorporated.
```
Comment (AndrewHaisting) [inline Lucas/Mexican/flourTortillas.md:10]: Excellent detail here 👍

**Review by AndrewHaisting — CHANGES_REQUESTED** [review]
Everything looks good here, I just have one optional comment about content, and one required change to make this commit history correct.

File: Lucas/Italian/sugo_di_pomodoro-marinara.md (hunk 1 above)
Comment (lukekivi) [reply Lucas/Italian/sugo_di_pomodoro-marinara.md:21]: I initially pushed the branch without committing changes to the branch. That may be the issue. So then I committed changes and re-pushed and here we are.

File: Lucas/Mexican/flourTortillas.md (hunk 2 above)
Comment (lukekivi) [reply Lucas/Mexican/flourTortillas.md:5]: Good point -- the original recipe says he prefers half and half veg - animal actually

Comment (lukekivi) [reply Lucas/Mexican/flourTortillas.md]: e05dee7ba3a9b8b4fef30d331474bd8ba50992d0 Yep sounds good, changed.

Comment (AndrewHaisting) [reply Lucas/Mexican/flourTortillas.md]: nice 👍

**Review by AndrewHaisting — APPROVED** [review]
🚀

**Pull request merged**